CMakeCommon.cmake

```diff
@@ -1,3 +1,7 @@
+if (CMAKE_VERSION VERSION_LESS "3.20.0")
+    message (FATAL_ERROR "This script now requires CMake 3.20.0 or newer")
+endif ()
```

Comment:
A version requirement must be stated using [cmake_minimum_required](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html). This will fail configuration if the minimum requirement is not met and will set the necessary policies up until the stated version.

But since this script is not a standalone entry point (CMakeLists.txt or a CMake script), this should be done in the associated entry point instead.